server/src/LobbyRoom.ts

```diff
@@ -0,0 +1,41 @@
+import { Client, Room } from 'colyseus';
+import { Player, playerState } from './Models';
+
+export type PlayersMap = { [id: string]: Player };
+
+export interface LobbyRoomState {
```

Comment:
in order to allow different types of players in a lobby (aka to add a team if needed),
`PlayerMap` should receive a template type that extends `Player`,
and then `LobbyRoomState` would receive it and save the `PlayersMap` with the correct player type.